belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssue.java

```diff
@@ -68,6 +72,12 @@ public InputValidationIssue(InEnum in, String name) {
         this.name = name;
     }
 
+    public InputValidationIssue(Input<?> input) {
```

Comment:
I don't think we really need this new constructor? It's only used in a test.

I'm not too happy with the pre-existing "in, name (, value)" constructors either, but probably not worth it to remove those. With these constructors, we facilitate the bad practice of not populating the type and title.

Reply:
> I don't think we really need this new constructor? It's only used in a test.
 Ok to remove it